autopilot/wallet/wallet.go

```diff
@@ -0,0 +1,107 @@
+package wallet
+
+import (
+	"context"
+	"fmt"
+	"sync"
+
+	"go.sia.tech/core/types"
+	cwallet "go.sia.tech/coreutils/wallet"
+	"go.sia.tech/renterd/alerts"
+	"go.sia.tech/renterd/api"
+	"go.sia.tech/renterd/autopilot/contractor"
+	"go.uber.org/zap"
+)
+
+type (
+	Bus interface {
+		Wallet(ctx context.Context) (api.WalletResponse, error)
+		WalletPending(ctx context.Context) (resp []cwallet.Event, err error)
+		WalletRedistribute(ctx context.Context, outputs int, amount types.Currency) (ids []types.TransactionID, err error)
+	}
+
+	Wallet interface {
```

Comment:
Same thing here. Best way to keep interfaces minimal while avoiding circular dependencies is to declare interfaces in the same package that consumes it. Like the `Wallet` interface in the `bus` package.